test/functions/generic/poi/reduce-user-record.chpl

```diff
@@ -9,7 +9,7 @@ record Foo {
 
 // We want these to be visible to the implementation of 'reduce'.
 operator :(const x: int, type t: Foo)  return new Foo(x);
-proc *(a: Foo, b: Foo)                 return new Foo(a.x*b.x);
+operator Foo.*(a: Foo, b: Foo)         return new Foo(a.x*b.x);
```

Comment:
If it works for this to be a non-method, I suggest switching to that because doing so will strengthen this illustration of POI semantics.